components/dropzone/stories/dropzone.stories.js

```diff
@@ -35,6 +34,10 @@ export default {
 		rootClass: "spectrum-DropZone",
 		isDragged: false,
 		isFilled: false,
+		customHeading: "Drag and drop your file",
+		customDescription: "Or, select a file from your computer.",
+		customLabel: "Browse files",
+		hasButton: true
```

Comment:
These controls are not showing up in a particular category: 

<img width="262" alt="Screenshot 2024-11-26 at 2 46 32 PM" src="https://github.com/user-attachments/assets/5b94b161-6886-4a57-ab14-006ccfc227e9">

I wonder if we can bring this approach over from `main`, where we use `...IllustratedMessage.argTypes` and that `label: Button.argTypes.label`: https://github.com/adobe/spectrum-css/blob/8382527c9d1aa0889829ef9868b609f3b018b9b3/components/dropzone/stories/dropzone.stories.js#L29

We could then just define those in the default args here instead (if we want it to say anything different from the default illustrated message heading or description): 
```
heading: "Drag and drop your file",
description: "Or, select a file from your computer.",
label: "Browse files",
hasButtons: true,
```

Then in the template, we could just use those same `heading`, `description` args for the `IllustratedMessage()`:

```
${IllustratedMessage({
			heading,
			description,
			customIllustration: customSvg,
		})}
```